### PR TITLE
fix build

### PR DESCRIPTION
**Pull request by felicio**
- add dependencies from monorepo's root
- remove Buffer from middleware
- use webpack alias
- use nextjs transpilePackages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,31 +3,36 @@
...
   },
   "dependencies": {
-    "@radix-ui/react-dialog": "^1.0.3",
-    "@radix-ui/react-navigation-menu": "^1.1.2",
-    "@react-spring/web": "^9.7.2",
-    "@scure/base": "^1.1.1",
     "@status-im/colors": "*",
     "@status-im/components": "*",
     "@status-im/icons": "*",
     "@status-im/js": "*",
```
Comment (felicio) [inline package.json:20]: awaits https://github.com/status-im/status-web/pull/432

File: tsconfig.json
Comment (felicio) [inline tsconfig.json:1]: follow-up: create a package like the one for eslint config

File: tsconfig.json [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
-      ]
+      "@/*": ["./src/*"]
+      // "@status-im/js/encode-url-data": [
+      //   "../../packages/status-js/src/utils/encode-url-data"
+      // ],
+      // "@status-im/js/encode-url-hash": [
+      //   "../../packages/status-js/src/utils/encode-url-hash"
+      // ]
+      // "@status-im/js": ["./node_modules/@status-im/js/dist"],
+      // "@status-im/components": ["./node_modules/@status-im/components/dist"]
```
Comment (felicio) [inline tsconfig.json:21]: ditto https://github.com/status-im/status-website/pull/1/files#r1245883913, see if still necessary after publish

**Review by prichodko — APPROVED** [review]
🚀

**Pull request merged**